<ui version="4.0" >
 <class>BasicDynamicProgramEditor</class>
 <widget class="QWidget" name="BasicDynamicProgramEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>250</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Size</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QCheckBox" name="sizeEnabled" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="labelSizeJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Jitter:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="KIntNumInput" name="sizeJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelSizeSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KisSensorSelector" name="sizeSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>211</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="labelSizeMinimum" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Minimum:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="KIntNumInput" name="sizeMinimum" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="value" >
          <number>100</number>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>200</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="labelSizeMaximum" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Maximum:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="KIntNumInput" name="sizeMaximum" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="value" >
          <number>100</number>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>200</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>79</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Angle</string>
      </attribute>
      <layout class="QGridLayout" >
       <item rowspan="2" row="0" column="0" colspan="3" >
        <widget class="QCheckBox" name="angleEnabled" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
       <item rowspan="2" row="1" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="labelAngleJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Jitter:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KIntNumInput" name="angleJitter" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="minimum" >
          <number>0</number>
         </property>
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="suffix" >
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="labelAngleSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Sensor:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="KisSensorSelector" name="angleSensor" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>79</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Scattering</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Scatter</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" colspan="3" >
           <widget class="QCheckBox" name="scatterEnabled" >
            <property name="text" >
             <string>enabled</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2" >
           <widget class="QLabel" name="labelScatterAmount" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Amount:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="KIntNumInput" name="scatterAmount" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>0</number>
            </property>
            <property name="maximum" >
             <number>1000</number>
            </property>
            <property name="suffix" >
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QLabel" name="labelScatterJitter" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Jitter:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="KIntNumInput" name="scatterJitter" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>0</number>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="suffix" >
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="labelScatterSensor" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Sensor:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2" >
           <widget class="KisSensorSelector" name="scatterSensor" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Count</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" colspan="2" >
           <widget class="QCheckBox" name="countEnabled" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>enabled</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelCountCount" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Count:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KIntNumInput" name="countCount" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>1</number>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="suffix" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="labelCountJitter" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Jitter:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="KIntNumInput" name="countJitter" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="minimum" >
             <number>0</number>
            </property>
            <property name="maximum" >
             <number>100</number>
            </property>
            <property name="suffix" >
             <string>%</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="labelCountSensor" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
            <property name="text" >
             <string>Sensor:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="KisSensorSelector" name="countSensor" >
            <property name="enabled" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>91</width>
           <height>113</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>153</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KisSensorSelector</class>
   <extends>QComboBox</extends>
   <header location="global" >kis_sensor_selector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
